seed/landing/urls.py

```diff
@@ -17,10 +18,6 @@
 
 urlpatterns = [
     url(r'^$', landing_page, name='landing_page'),
-    url(r'^accounts/create/$', create_account, name='create_account'),
-    url(r'^account_activation_sent/$', account_activation_sent, name='account_activation_sent'),
-    url(r'^activate/(?P<uidb64>[0-9A-Za-z_\-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/$',
```

Comment:
Still reviewing the code changes, and I haven't tested yet... but I think this would prevent _anyone_ from activating an account even if the invite/activation came from an existing user trying to invite a new user to join an existing org.

Reply:
I was able to add users to organizations locally.  This logic flow is actually separate from the invitation and account setup flow from adding users to organizations.  I could be missing something though!

Reply:
My bad! I didn't verify my assumption that it was the same flow! :smh:

Yeah this does seem to still allow invitations from existing admins to go through and sign ups to be successful 👍🏽